test/Microsoft.AspNetCore.SignalR.Client.Tests/ConnectionTests.cs

```diff
@@ -406,9 +410,9 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
             await connection.StartAsync();
             channel.Output.TryWrite(Array.Empty<byte>());
             channel.Output.TryWrite(Array.Empty<byte>());
-            await allowDisposeTcs.Task.OrTimeout();
```

Comment:
The idea behind this test now is that we would block in the user code and will check if is HttpConnection continues to consume incoming messages. If it did not it would mean the user's code blocked the SignalR thread.
@mikaelm12 - please add a comment explaining how this test is supposed to work. This is probably the most convoluted test we currently have.